recipes/ixwebsocket/all/conanfile.py

```diff
@@ -0,0 +1,146 @@
+                                                          and not self.canUseOpenSSL() or self.options.use_mbed_tls):
+            self.requires.add("mbedtls/2.6.1")
+
+        if self.settings.os == "Macos" and not self.options.use_openssl and not self.options.use_mbed_tls:
+            # Required
+             self.cpp_info.frameworks = [ 'Security' ]
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("IXWebSocket-" + self.version, "sources")
+
+    def addLibrary(self, keyBase, includeName, cmake, plural=False):
+        cmake.definitions[keyBase +
+                          "_LIBRARY"] = self.deps_cpp_info[includeName].rootpath
+        includePath = keyBase + "_INCLUDE_DIR"
+        if plural == True:
+            includePath += "S"
+        cmake.definitions[includePath] = os.path.join(
+            self.deps_cpp_info[includeName].rootpath, self.deps_cpp_info[includeName].includedirs[0])
+
+    def configure_cmake(self):
+        cmake = CMake(self)
+
+        # User-selectable options
+        cmake.definitions["USE_TLS"] = self.options.use_tls
+        cmake.definitions["USE_MBED_TLS"] = self.options.use_mbed_tls
+        cmake.definitions["USE_WS"] = self.options.use_ws
+        cmake.definitions["USE_VENDORED_THIRD_PARTY"] = self.options.use_vendored_third_party
+
+        # Library linking
+        if (self.options.use_tls and not self.options.use_mbed_tls and not self.settings.os == "Windows" and not self.settings.os == "Macos"):
+            os.environ['OPENSSL_ROOT_DIR'] = self.deps_cpp_info["openssl"].rootpath
+        self.addLibrary("ZLIB", "zlib", cmake)
+        if not self.options.use_vendored_third_party and (self.options.use_mbed_tls or self.options.use_tls and self.settings.os == "Windows"):
+            self.addLibrary("MBEDTLS", "mbedtls", cmake, True)
+            self.addLibrary("MBEDCRYPTO", "mbedtls", cmake, True)
+            self.addLibrary("MBEDX509", "mbedtls", cmake, True)
+        cmake.configure()
+        return cmake
+
+    def build(self):
+        cmake = self.configure_cmake()
+        cmake.build()
+        cmake.install()
+
+    def package(self):
+        # Include package license
+        self.copy("LICENSE.txt", dst="licenses", src="sources")
+
+        # Include binaries and headers
+        self.copy("*.h", dst="include", src="include")
+        self.copy("*.lib", dst="lib", keep_path=False)
+        self.copy("*.dll", dst="bin", keep_path=False)
+        self.copy("*.so", dst="lib", keep_path=False)
+        self.copy("*.dylib", dst="lib", keep_path=False)
+        self.copy("*.a", dst="lib", keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.options.use_tls and self.settings.os == "Windows":
+            # Include linking with the websocket
+            self.cpp_info.libs += ["Ws2_32"]
```

Comment:
You can use `.append("Ws_32")`

Reply:
Use `system_libs` as that one is a system library, not something packaged by this recipe